apps/web/app/modules/Courses/Lesson/LessonItems/Presentation.tsx

```diff
@@ -35,7 +37,7 @@ export default function Presentation({
     const isInViewport = intersection && intersection.intersectionRatio === 1;
 
     const loadTimeout = setTimeout(() => {
-      if (isInViewport && !isCompleted) {
+      if (isInViewport && !isCompleted && !isAdmin) {
```

Comment:
`React Hook useEffect has a missing dependency: 'isAdmin'. Either include it or remove the dependency array` - is it intended?